publication-commons/src/test/resources/features/AnonymousAccessRights.feature

```diff
@@ -0,0 +1,66 @@
+    And the UPDATE method requires a non empty publication ID
+    And the DynamoDBPublicationService has an DELETE method
+    And the DELETE method requires a user with non empty username
+    And the DELETE method requires a non empty publication ID
+    And the DynamoDBPublicationService has a CREATE method
+    And the CREATE method requires a user with non empty username
+    And the CREATE method requires a non empty publication object
+
+  Scenario: Anonymous user reads published material
+    Given a publication with ID "PubId"
+    And the publication "PubId" has status PUBLISHED
+    When READ is called on behalf of the Anonymous user for the publication "PubId"
+    Then READ returns the publication "PubId"
+
+  Scenario: Anonymous user tries to read unpublished material
+    Given a publication with ID "PubId"
+    And the publication "PubId" does not have status PUBLISHED
+    When READ is called on behalf of the Anonymous user for the publication "PubId"
+    Then READ returns a response that the item with ID "PubID" was not found.
+
+  Scenario: Anonymous user tries to read non existing
+    Given a non-existing publication ID "nonExistingID"
+    When READ is called on behalf of the Anonymous user for the publication "nonExistingID"
+    Then READ returns a response that the item with ID "nonExistingID" was not found.
+
+  Scenario: Anonymous user tries to create published material
+    Given a publication PublicationA
+    When CREATE is called on behalf of the Anonymous user for the publication object PublicationA
+    Then CREATE returns a response that this action is not allowed
+
+  Scenario Outline: Anonymous user tries to update/delete material
+    Given a publication with ID "PubId"
+    When <non-read-action> is called on behalf of the Anonymous user for the publication "PubId"
+    Then <non-read-action> returns a response that this action is not allowed
+
+    Examples:
+      | non-read-action |
+      | UPDATE          |
+      | DELETE          |
+
+  Scenario: Anonymous users can see published material when they list publications
+    Given that DynamoDBPublicationService has a LIST method
+    And that LIST requires a user with non empty username
+    And that LIST requires a publication owner's username
+    When LIST is called on behalf of the Anonymous user for the user "theCreator"
+    Then LIST returns all published publications whose owner is the user "theCreator"
+
+  Scenario: Anonymous users can NOT see unpublished material when they list publications
+    Given that DynamoDBPublicationService has a LIST method
+    And that LIST requires a user with non empty username
+    And that LIST requires a publication owner's username
+    And publication "PubID" does not have status PUBLISHED.
+    And the owner of "PubID" is the user with username "theCreator"
+    When LIST is called on behalf of the Anonymous user for the user "theCreator"
+    Then the "PubId" publication is NOT in the result-set
```

Comment:
Given the "published publications" in previous senario - is this senario needed?

Reply:
The previous scenario asserts that all published publications of a creator are visible. If the method returns everything (published and unpublished) it satisfies the above scenario as it has returned the published. This scenario asserts that the unpublished material is not visible to the anonymous user.